ListOfBooks.xcodeproj/project.pbxproj

```diff
@@ -13,12 +13,15 @@
 		1C325B502ADDA69D007593D9 /* List.swift in Sources */ = {isa = PBXBuildFile; fileRef = 1C325B4F2ADDA69D007593D9 /* List.swift */; };
 		1C325B592ADEF35B007593D9 /* ListCollectionViewCell.swift in Sources */ = {isa = PBXBuildFile; fileRef = 1C325B572ADEF35B007593D9 /* ListCollectionViewCell.swift */; };
 		1C325B5A2ADEF35B007593D9 /* ListCollectionViewCell.xib in Resources */ = {isa = PBXBuildFile; fileRef = 1C325B582ADEF35B007593D9 /* ListCollectionViewCell.xib */; };
+		1CA4C7AE2AE68377002CB107 /* Localized string.swift in Sources */ = {isa = PBXBuildFile; fileRef = 1CA4C7AD2AE68377002CB107 /* Localized string.swift */; };
```

Comment:
<img width="497" alt="Screenshot 2023-11-27 at 17 12 14" src="https://github.com/potemkin123/ListOfBooks/assets/14887101/8ef0345c-5e5f-4c52-a11f-0dd3d34019d5">

Move all models to separate files